db/kafka/migrations/V1.0.0.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Logging topics
```

Comment:
Provide a upgrade migration to V1.0.0.
Using same convention as flyway database migration